src/pages/UserHome.jsx

```diff
@@ -41,12 +44,29 @@ const useStyles = makeStyles((theme) => ({
   noScholarshipsBox: {
     textAlign: 'center',
   },
+  progress: {
+    display: 'block',
+  },
+  loadMoreButton: {
+    margin: theme.spacing(3, 0),
+  },
 }));
 
 export default function UserHome() {
   const classes = useStyles();
+  const [scholarships, setScholarships] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState();
   const user = firebase.auth().currentUser;
 
+  useEffect(() => {
+    Scholarships.list({ authorId: user.uid })
+      .then((results) => setScholarships(results))
```

Comment:
My [previous comment](https://github.com/beyondhb1079/s4us/pull/397#discussion_r584230211) is still relevant.